content/webapp/services/prismic/transformers/index.ts

```diff
@@ -128,12 +134,17 @@ export function transformTimestamp(field: TimestampField): Date | undefined {
 }
 
 // Prismic often returns empty RichText fields as `[]`, this filters them out
-export function transformRichTextField(
-  field: RichTextField
-): HTMLString | undefined {
+export function asHtmlString(field: RichTextField): HTMLString | undefined {
```

Comment:
I find the name of this function (which goes back to the name of the type it returns) a bit confusing.
Seeing as it's an array of [HTMLStringBlock types](https://github.com/wellcomecollection/wellcomecollection.org/blob/8d400fbf759ea651c04900abb89128bfea3e3201/common/services/prismic/types.ts#L38). Could we call it something more like asHTMLStringBlocks and rename HTMLString to HTMLStringBlocks or similar?

Reply:
Good point. My plan is to ditch our HTMLStringBlock type entirely, because the Prismic types have a RichTextField which is almost the same – I'm renamed this to `asRichText`, as that's where this is eventually going.

Reply:
I've just realised you wrote that in the the top comment :) asRichText is much nicer, thanks